Intersect.Client/Entities/Entity.cs

```diff
@@ -1059,33 +1059,35 @@ public virtual void Draw()
             }
         }
 
-        private int NormalSpriteAnimationFrame(int frame)
+        private int NormalSpriteAnimationFrame()
```

Comment:
I think I expected the input parameter to be used for something, but since there are no longer any input parameters change this into a property.